docker/opts/opts_unix.go

```diff
@@ -20,3 +20,12 @@ func MountParser(mount string) (source, destination string, err error) {
 	}
 	return "", "", errors.Wrap(fmt.Errorf("invalid mount format: got %s, expected <src>:<dst>", mount), "")
 }
+
+// VolumeParser parsed volume path.
+func VolumeParser(volume string) (source, target string, err error) {
+	st := strings.Split(volume, ":")
+	if len(st) == 3 {
+		return st[1], st[2], nil
+	}
+	return "", "", errors.Wrap(fmt.Errorf("invalid volume format: got %s, expected <drive>:<src>:<trgt>", volume), "")
```

Comment:
Looks like this got copied from `opts_windows.go`. I think it should just take in `<src>:<dst>` (split with len 2)

It'd be great if we could allow the parser to accept `<src>:<dst>:ro` as a valid input and then pass `ro` as a bool indicating if the volume is read only